Dockerfile

```diff
@@ -264,35 +249,37 @@ RUN ln -sfv /go/src/github.com/docker/docker/.bashrc ~/.bashrc
 RUN echo "source /usr/share/bash-completion/bash_completion" >> /etc/bash.bashrc
 RUN ln -s /usr/local/completion/bash/docker /etc/bash_completion.d/docker
 RUN ldconfig
-# This should only install packages that are specifically needed for the dev environment and nothing else
-# Do you really need to add another package here? Can it be done in a different build stage?
```

Comment:
Again, I'd keep this comment to not diverge from upstream. (This one is present even on Moby `master`.)

Reply:
@lmbarros yes, we can keep the comments.